### PR TITLE
Update icons for relevant and irrelevant records

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
I think the 'X' might be confusing because it is typically used for closing something. Therefore, I propose to use a thumb-down icon.

### CODE REVIEW

File: asreview/webapp/src/Components/HistoryDialog.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { makeStyles } from '@material-ui/core/styles';
 
 import ArrowBackIcon from '@material-ui/icons/ArrowBack';
 import FavoriteIcon from '@material-ui/icons/Favorite';
-import CloseIcon from '@material-ui/icons/Close';
+import ThumbDownIcon from '@material-ui/icons/ThumbDown';
```
Comment (J535D165) [inline asreview/webapp/src/Components/HistoryDialog.js:21]: Interesting idea! 

If we are changing it here, then we should also change it in the review screen (and Prior Knowledge screen?).

Comment (Rensvandeschoot) [reply asreview/webapp/src/Components/HistoryDialog.js:21]: I think it makes sense...

Comment (terrymyc) [reply asreview/webapp/src/Components/HistoryDialog.js:21]: I'll make a screenshot for preview!

**Comment by Rensvandeschoot** [comment]
I changed the heading of the pop-up screen back to Details and i will open a separate PR for it.

**Comment by terrymyc** [comment]
image

image

**Comment by Rensvandeschoot** [comment]
o nice! looking good!

@J535D165 I propose to integrate this in a future release (v0.17 / 18?) because we have to also adjust the documentation

**Comment by J535D165** [comment]
This requires quite some updates on the documentation. Please use this PR for documentation updates.

**Comment by Rensvandeschoot** [comment]
@J535D165 I changed the icon in the decision bar. It is not used in the prior knowledge phase.
@terrymyc I also update the icon in `HistoryListCard.js`

**Comment by Rensvandeschoot** [comment]
New screenshots are need on lines 33/34 + 111/112: https://github.com/asreview/asreview/blob/master/docs/source/features/screening.rst

and lines 88/89
https://github.com/asreview/asreview/blob/master/docs/source/lab/oracle.rst

**Comment by MartijnUX** [comment]
We could consider to change the hart as well to a finger up icon. In that way it is clear that it is a negative/positive. See example.
image

**Comment by MartijnUX** [comment]
@terrymyc Is it possible to put the icon before the text? See example 
image

**Comment by Rensvandeschoot** [comment]
I am fine with changing the hart in a thumbs-up. I think the colors should indeed be green and gray, like above.

**Comment by terrymyc** [comment]
> Is it possible to put the icon before the text? See example

@MartijnUX Yes, the icon can be put before.

**Comment by Rensvandeschoot** [comment]
@terrymyc I have added the 'thumbsup' icon, can you see if you can make it green? and update the order as requested by @MartijnUX? many thanks!

**Comment by J535D165** [comment]
After seeing the updates and screenshots, I'm not directly convinced about this change. On the other hand, this PR convinces me that we should consider changes. It's a great discussion and team effort. 

The thumbs-up and thumbs-down approach gives it a bit of a like-dislike feeling. As with social media. That's not in line with the things we do. It also doesn't translate to relevant or irrelevant for me. Not a big problem when labels (relevant and irrelevant) are available. However, it might be confusing when used standalone. Do you like the record, or do you want to mark the item as relevant. 

My main concern is the extensibility. We are planning/ thinking about options like skipping, postponing, and multilabel decisions. The thumbs-up and thumbs-down gestures are hard to extend with something like skipping. I think we should also think about the colors. It makes sense to make relevant green, however, it makes more sense to make irrelevant records red. This leaves grey records for postponing/unlabeled/skipping for example. 

Material turns out to have a component that might fit our purpose. It is named Chip https://material-ui.com/components/chips/#chip. You find this component in your email box for tags, or when filling in keywords when submitting a paper for example.

The following example shows how a chip can look like. I decided to use letters instead of icons. 

Screenshot 2021-01-11 at 19 55 48
Screenshot 2021-01-11 at 19 54 22

Bonus

image

I'm still a bit struggling with the overview page. The following page is NOT good imo, but I'm sharing to facilitate the discussion. 

image

**Comment by terrymyc** [comment]
@J535D165 I like the outlined chips! Maybe we went too far with the `thumb` idea. The concern in the first place was a cross may be confusing on the details (overview) page. I suggest replacing the cross and heart with chips only on the left bottom of the overview page.

**Comment by Rensvandeschoot** [comment]
ok! let's work on this for V1.0

**Pull request closed without merging**